From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 02 Apr 2017 02:06:06 +0100
Subject: mmc: Avoid ABI change for "mmc: core: Annotate cmd_hdr as __le32"
Forwarded: not-needed

Commit 3f2d26643595973e835e8356ea90c7c15cb1b0f1 changed the type alias
used to declare mmc_packed::cmd_hdr from __le32 to u32, but the field
apparently already contained little-endian words so this was not an ABI
change.  Hide it from genksyms.

---

--- a/drivers/mmc/card/queue.h
+++ b/drivers/mmc/card/queue.h
@@ -24,7 +24,11 @@ enum mmc_packed_type {
 
 struct mmc_packed {
 	struct list_head	list;
+#ifdef __GENKSYMS__
+	u32			cmd_hdr[1024];
+#else
 	__le32			cmd_hdr[1024];
+#endif
 	unsigned int		blocks;
 	u8			nr_entries;
 	u8			retries;
